src/Serializer.php

```diff
@@ -48,7 +48,7 @@ public function serialize(SegmentInterface ...$segments): string
         $message .= $this->characters->getDataSeparator();
         $message .= $this->characters->getDecimalPoint();
         $message .= $this->characters->getEscapeCharacter();
-        $message .= " ";
+        $message .= $this->characters->getReservedSpace();
```

Comment:
I think the UN/EDIFACT specification says that this character is used for
> Blank/space, reserved for later applications 

And that is what the "reservedSpace" property from Characters is meant for, doesnt it? This: https://github.com/metroplex-systems/edifact/blob/master/src/Control/Characters.php#L42

So for my client this character is defined as "Multi-Separator" (whatever that is) and that has to be a "*". Therefore I must be able to modify the reserved character.

Reply:
Ah ok, I think the spec means that it's reserved for later use by them, rather than individual applications/clients using it for different purposes. So I'd warn your client about that multi separator usage. But yes I'll get it fixed and released shortly for you

Reply:
Cool thanks man!